packages/veritone-react-common/src/components/IngestionJobs/NullState/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import withMuiThemeProvider from 'helpers/withMuiThemeProvider';
+import Button from '@material-ui/core/Button';
+import { func } from 'prop-types';
+
+import NullstateImage from 'images/cms-ingestion-jobs-null.svg';
+import styles from './styles.scss';
+
+@withMuiThemeProvider
+export default class IngestionJobNullstate extends React.Component {
```

Comment:
Reuse the same component that you created in the tile view PR.

Reply:
fixed